NITC M.Tech Industrial Power and Automation 2023 S1-S2

MaryamMohamedYahya opened the pull request:
- **Syllabus Added:** Core courses offered under the scheme of M.Tech in Industrial Power and Automation from 2023 admission at National Institute of Technology Calicut
- **Files Added**:
<img width="805" height="587" alt="M Tech Industrial Power and Automation" src="https://github.com/user-attachments/assets/ba5801f5-7310-4f22-bfd2-beefd2a80092" />

9sreerag7 commented:
Hi @MaryamMohamedYahya  thanks for the updates! 

I noticed an issue with the branch name in the YAML. Since you’ve also updated the folder structure, please change the branch name from:
m-tech-industrial-power-and-automation
to:
industrial-power-and-automation
This will help keep the naming consistent.

MaryamMohamedYahya commented:
> I noticed an issue with the branch name in the YAML. Since you’ve also updated the folder structure, please change the branch name from: m-tech-industrial-power-and-automation to: industrial-power-and-automation This will help keep the naming consistent.

Thank you! 
Removed m-tech from the YAML Frontmatter. Please review @9sreerag7.
